notify/pushover/pushover.go

```diff
@@ -75,7 +74,8 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	}
 	data := notify.GetTemplateData(ctx, n.tmpl, as, n.logger)
 
-	level.Debug(n.logger).Log("incident", key)
+	// @tjhop: should this use `group` for the keyval like most other notify implementations?
+	n.logger.Debug("extracted group key", "incident", key)
```

Comment:
Agreed. I went with `group_key` in most places for sake of _something_ to key the log message on since most of the previous calls were bare; the pushover implementation already used the `incident` key, so I left if for consistency in that regard. 

If there's a preference what to align on, we can definitely circle back